OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -456,7 +456,7 @@ public bool CanExistInCell(CPos cell)
 
 		public bool CanEnterCell(CPos cell, Actor ignoreActor = null, bool checkTransientActors = true)
 		{
-			return Info.CanEnterCell(self.World, self, cell, ignoreActor, checkTransientActors);
+			return Info.CanEnterCell(self.World, self, cell, ToSubCell, ignoreActor, checkTransientActors);
```

Comment:
What happens if `ToSubCell` is blocked, but the other four are available?

Reply:
Certainly this case won't happen, since the `CellFlag.HasFreeSpace` flag will be set, letting `CanMoveFreelyInto` return true.

Reply:
A comment was added.